Sources/SolToBoogieTest/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿
+namespace SolToBoogieTest
+{
+    using System;
+    using System.IO;
+    using System.Runtime.InteropServices;
+    using Microsoft.Extensions.Logging;
+
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            if (args.Length != 2)
+            {
+                Console.WriteLine("Usage: SolToBoogieTest <path-to-corral> <workingdir>");
+                return;
+            }
+
+            string corralPath = args[0];
+            string workingDirectory = args[1];
+            string solcName = GetSolcNameByOSPlatform();
+            string solcPath = Path.Combine(workingDirectory, "Tool", solcName);
+            string testDir = Path.Combine(workingDirectory, "Test", "regressions");
+            string configDir = Path.Combine(workingDirectory, "Test", "config");

```

Comment:
Why do you need the configDir?

Reply:
The configDir is used to store the configurations for running Corral, and these configurations could be dependent on each benchmark. e.g., each benchmarks's "CorralEntry_ " is different.